mm2src/coins/rpc_command/tendermint/staking.rs

```diff
@@ -0,0 +1,149 @@
+use common::{HttpStatusCode, PagingOptions, StatusCode};
+use cosmrs::staking::{Commission, Description, Validator};
+use mm2_core::mm_ctx::MmArc;
+use mm2_err_handle::prelude::MmError;
+
+use crate::{lp_coinfind_or_err, tendermint::TendermintCoinRpcError, MmCoinEnum};
+
+/// Represents current status of the validator.
+#[derive(Default, Deserialize)]
+pub(crate) enum ValidatorStatus {
+    All,
+    /// Validator is in the active set and participates in consensus.
+    #[default]
+    Bonded,
+    /// Validator is not in the active set and does not participate in consensus.
+    /// Accordingly, they do not receive rewards and cannot be slashed.
+    /// It is possible to delegate tokens to a validator in this state.
```

Comment:
nit: `/// It is still possible to delegate ...` (addition of `still`)

Reply:
idk when i read the doc comments here for the first time they kinda gave me the intuition that `it is possible to delegate` to `Unbonded` validators and only `Unbonded` validators. but it's clearer for me now as i read it again.

demoted to a nitty^2 nit.